src/bin/fullnode-config.rs

```diff
@@ -67,9 +70,12 @@ fn main() {
     };
     let pkcs8 = read_pkcs8(id_path).expect("client keypair");
 
+    let rnd = SystemRandom::new();
+    let vote_account_pkcs8 = Ed25519KeyPair::generate_pkcs8(&rnd).unwrap();
+
     // we need all the receiving sockets to be bound within the expected
     // port range that we open on aws
-    let config = Config::new(&bind_addr, pkcs8);
+    let config = Config::new(&bind_addr, pkcs8, vote_account_pkcs8.to_vec());
```

Comment:
The vote account keypair is saved in the fullnode config so the node doesn't need to recreate the vote account on a restart.